### DISCUSSION TITLE
所有歌曲都无法播放

### DISCUSSION THREAD

**Discussion by alya0408** [lyswhut/lx-music-desktop#1638] [General]
为什么最新版的也用不了了，所有歌曲都无法播放，网络这些都是好的，代理也关了，接口也换了，所有源都用不了，我看公告说由于腾讯的投诉，暂停下载这些服务，是因为这个问题吗，那为什么用网易的源也不行，只是播放都不可以。

**Comment by ScottYuan09** [comment]
见Issue #5

**Comment by qushulan** [comment]
我也是用不了

**Reply by Master-20702** [reply]
一样，我也是用不了

**Reply by ZMLM123321** [reply]
我也是

**Reply by Master-20702** [reply]
我换个源就可以用了

**Comment by echo-fannnnnn** [comment]
用不了了诶，还麻烦作者辛苦下，修复一下吧，谢谢啦！

**Comment by ConmyCao** [comment]
大佬请修复一下吧~~~ 基本凉了。

**Comment by Moonwing2020** [comment]
是不是跑路了啊？

**Comment by aectn** [comment]
换这个音源，可以在LX里面改
https://github.com/OneCodeMonkey/music-sources
